Shift indentation to avoid the "misleading indentation" warning
from LLVM10. I admit, that the original author's intent escapes
me here, so I cannot tell, if we have an actual bug remaining
in this code.

--- fpx/ptil_fpx.cpp	2013-09-02 11:45:00.000000000 -0400
+++ fpx/ptil_fpx.cpp	2020-03-17 18:11:00.425447000 -0400
@@ -1200,7 +1200,7 @@
           pixelsSpace = tmpPixelSpace;
         }
-        imageParam->GetContrast( &contrastValue);
-        Contrast( contrastValue, pixelsSpace, pixels, width * height);
-      }
+      imageParam->GetContrast( &contrastValue);
+      Contrast( contrastValue, pixelsSpace, pixels, width * height);
+    }
   } 
 
The variable No_JPEG_Header_Flag is unused. But its mere presence may
cause a "duplicate symbol" error from linker, when it tries to link
together object files compiled after #include-ing the ejpeg.h header.

--- jpeg/ejpeg.h	2013-09-02 11:45:00.000000000 -0400
+++ jpeg/ejpeg.h	2020-09-07 10:50:07.600375000 -0400
@@ -31,5 +31,2 @@
 #define IMG_FORMAT_1111 0x04   /* Four-component non-interleaved image */
 #define IMG_FORMAT_411  0x05   /* Three-component 4:1:1  image */
-
-JPEGEXPORT
-int No_JPEG_Header_Flag;
